lib/paper_trail/has_paper_trail.rb

```diff
@@ -306,6 +303,14 @@ def source_version
         send self.class.version_association_name
       end
 
+      def preceding_version
+        if source_version
+          source_version.previous
+        else
+          send(self.class.versions_association_name).last
+        end
+      end
```

Comment:
I know I said I'd prefer to extract private methods, but we can't do that here because we already have a problem with polluting people's AR models with a ton of methods, and I don't want to make that problem worse. See discussion in https://github.com/airblade/paper_trail/pull/719